Utilities/Thread.cpp

```diff
@@ -1404,7 +1404,7 @@ static LONG exception_handler(PEXCEPTION_POINTERS pExp)
 
 	if (pExp->ExceptionRecord->ExceptionCode == EXCEPTION_ACCESS_VIOLATION && addr64 < 0x100000000ull)
 	{
-		if (thread_ctrl::get_current() && handle_access_violation((u32)addr64, is_writing, pExp->ContextRecord))
+		if (handle_access_violation((u32)addr64, is_writing, pExp->ContextRecord))
```

Comment:
No. Don't.

Reply:
This allows rsx mem faults on threads that don't have a thread_ctrl such as the rsx debugger and rsx replay. The other solution is to "clean" dirty rsx mem sections before touching it. @ruipin, what do you think?

Reply:
Neither is a solution, debugger shouldn't trigger page faults at all.

Reply:
Fixing debuggers is a lot of work anyway, let's merge only rsx-capture fix in this PR.